Atomic/atomic.py

```diff
@@ -362,7 +362,9 @@ def gen_cmd(self, cargs):
             return " ".join(args)
 
     def get_fq_name(self, image_info):
-        if len(image_info['RepoTags']) > 1:
+        if not image_info['RepoTags']:
+            return None
+        elif len(image_info['RepoTags']) > 1:
```

Comment:
If it doesn't exist it will default to:
`else:
    return image_info['RepoTags'][0]`
Which will also cause the issue.